scripts/logging-functions.js

```diff
@@ -0,0 +1,4 @@
+function test(x,y){
```

Comment:
Use `es6`arrow function syntax here, also, change the input parameters to the function to something more wholesome. maybe `inputValue1` and `inputValue2` should do. Or something else you choose.